src/components/FacilityAddressModal.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <ion-header>
+    <ion-toolbar>
+      <ion-buttons slot="start">
+        <ion-button @click="closeModal()">
+          <ion-icon slot="icon-only" :icon="closeOutline" />
+        </ion-button>
+      </ion-buttons>
+      <ion-title>{{ translate("Address") }}</ion-title>
+    </ion-toolbar>
+  </ion-header>
+
+  <ion-content>
+    <form @keyup.enter="saveAddress">
+      <ion-item>
+        <ion-label>{{ translate("Address line 1") }} <ion-text color="danger">*</ion-text></ion-label>
+        <ion-input v-model="address.address1" slot="end" />
+      </ion-item>
+      <ion-item class="ion-margin-bottom">
+        <ion-label>{{ translate("Address line 2") }}</ion-label>
+        <ion-input v-model="address.address2" slot="end" />
+      </ion-item>
+      <ion-item>
+        <ion-label>{{ translate("City") }} <ion-text color="danger">*</ion-text></ion-label>
+        <ion-input v-model="address.city" slot="end" />
+      </ion-item>
+      <ion-item @keyup.enter.stop>
+        <ion-label>{{ translate("Country") }}</ion-label>
+        <ion-select interface="popover" :placeholder="translate('Select')" @ionChange="updateState($event)" v-model="address.countryGeoId">
+          <ion-select-option v-for="(country, index) in countries" :key="index" :value="country.geoId">{{ country.geoName }}</ion-select-option>
+        </ion-select>
+      </ion-item>
+      <ion-item @keyup.enter.stop>
+        <ion-label>{{ translate("State") }}</ion-label>
+        <ion-select interface="popover" :placeholder="translate('Select')" v-model="address.stateGeoId">
+          <ion-select-option v-for="(state, index) in states[address.countryGeoId]" :key="index" :value="state.geoId">{{ state.geoName }}</ion-select-option>
```

Comment:
```suggestion
          <ion-select-option v-for="state in states[address.countryGeoId]" :key="state.geoId" :value="state.geoId">{{ state.geoName }}</ion-select-option>
```
Make same changes for country ion-select.